lib/presentation/auth/widgets/enter_username.dart

```diff
@@ -100,6 +171,7 @@ class _EnterUserNameState extends State<EnterUserName> {
                     onTap: () {
                       if (_isNameValid && state is! AwaitingUsernameUpdate) {
                         FocusScope.of(context).unfocus();
+                        _username = _firstName! + " " + _lastName!;
                         context
                             .read<AuthBloc>()
                             .add(AuthEvent.updateUsername(_username!));
```

Comment:
This needs to be changed, it needs to send firstName and lastName to the API separated, not as only firstName

Reply:
Kindly advise, `updateUsername(String username)` has only a single string parameter